R/target_market_share.R

```diff
@@ -401,3 +407,49 @@ aggregate_by_loan_id <- function(data) {
     ) %>%
     ungroup()
 }
+
+check_unexpected_columns <- function(data) {
```

Comment:
To make this helper more reusable I suggest passing `valid_columns` as an argument. This way the useful logic is not mixed with the low-level detail of which specific columns are valid.